iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/relative_pointer.hpp

```diff
@@ -36,22 +36,24 @@ class RelativePointer : public BaseRelativePointer
     using const_ptr_t = const T*;
     using ptr_t = T*;
 
+    using BaseRelativePointer::BaseRelativePointer;
+
     /// @brief default constructs a RelativePointer as a logical nullptr
     RelativePointer() noexcept = default;
 
     /// @brief constructs a RelativePointer pointing to the same pointee as ptr in a segment identified by id
     /// @param[in] ptr the pointer whose pointee shall be the same for this
     /// @param[in] id is the unique id of the segment
-    RelativePointer(ptr_t ptr, id_t id) noexcept;
+    //    RelativePointer(ptr_t ptr, id_t id) noexcept;
```

Comment:
Rule A12-1-6 (required, implementation, automated)
Derived classes that do not need further explicit initialization and require
all the constructors from the base class shall use inheriting constructors.